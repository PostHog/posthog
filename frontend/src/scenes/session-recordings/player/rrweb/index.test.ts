import { CorsPlugin } from '.'

describe('CorsPlugin', () => {
    it.each([
        `@font-face { font-display: fallback; font-family: "Roboto Condensed"; font-weight: 400; font-style: normal; src: url("https://posthog.com/assets/fonts/roboto/roboto_condensed_reg-webfont.woff2?11012022") format("woff2"), url("https://posthog.com/assets/fonts/roboto/roboto_condensed_reg-webfont.woff?11012022")`,
        `url("https://app.posthog.com/fonts/my-font.woff2")`,
    ])('should replace font urls', (content: string) => {
        expect(CorsPlugin._replaceFontURLs(content)).toMatchSnapshot()
    })
})
