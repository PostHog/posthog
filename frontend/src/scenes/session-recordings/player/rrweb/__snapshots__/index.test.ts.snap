// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CorsPlugin should replace font urls 1`] = `"@font-face { font-display: fallback; font-family: "Roboto Condensed"; font-weight: 400; font-style: normal; src: url("https://replay-proxy.posthog.com?url=https://posthog.com/assets/fonts/roboto/roboto_condensed_reg-webfont.woff2?11012022") format("woff2"), url("https://replay-proxy.posthog.com?url=https://posthog.com/assets/fonts/roboto/roboto_condensed_reg-webfont.woff?11012022")"`;

exports[`CorsPlugin should replace font urls 2`] = `"url("https://replay-proxy.posthog.com?url=https://app.posthog.com/fonts/my-font.woff2")"`;
